api/v1alpha1/virtualmachine_conversion.go

```diff
@@ -902,6 +1040,11 @@ func (src *VirtualMachine) ConvertTo(dstRaw conversion.Hub) error {
 	// Manually restore data.
 	restored := &v1alpha2.VirtualMachine{}
 	if ok, err := utilconversion.UnmarshalData(src, restored); err != nil || !ok {
+		// If there is no v1a2 object to restore, this was originally a v1a1 VM
+		// Make sure the v1a1 PreReq conditions are translated over to v1a2 correctly
+		newConditions := convert_v1alpha1_PreReqsReadyCondition_to_v1alpha2_Conditions(dst)
```

Comment:
Not a big deal but I'd prb just remove the temp var and assign it directly 'cause it might not actually be any new conditions.